cmd/prometheus/main.go

```diff
@@ -174,11 +183,13 @@ type Reloadable interface {
 
 func reloadConfig(filename string, rls ...Reloadable) bool {
 	log.Infof("Loading configuration file %s", filename)
+	configAttempts.WithLabelValues("attempted").Add(1)
```

Comment:
This isn't a partition over the space, this should be two metrics - attempts and successes.

The last unixtime a config reload succeeded would probably also be useful.